pkg/distro/fedora/distro.go

```diff
@@ -572,6 +543,25 @@ func newDistro(version int) distro.Distro {
 	openstackImgType := qcow2ImgType
 	openstackImgType.name = "openstack"
 
+	vhdImgType := qcow2ImgType
```

Comment:
What's the benefit of making a copy the `qcow2ImgType` if most of its properties are overridden anyway? :thinking: 

I think that this approach introduces unnecessary surprises when the `qcow2` image gets modified, with very little benefit...

Reply:
Well, it's mimicking how these images are defined in the Fedora cloud kickstarts. The Azure image inherits from the qcow2 one. 

> I think that this approach introduces unnecessary surprises when the qcow2 image gets modified, with very little benefit...

...which matches our "upstream".

Reply:
Fair enough, although personally I still don't think that doing so brings much value.